gubernator/view_build.py

```diff
@@ -52,10 +52,9 @@ def handle_test(self, child, filename, name_prefix=''):
         elif child.find('failure') is not None:
             time = float(child.attrib['time'])
             out = []
-            for param in child.findall('system-out'):
-                out.append(param.text)
-            for param in child.findall('system-err'):
-                out.append(param.text)
+            for param in child.findall('system-out') + child.findall('system-err'):
+                if param.text is not None:
+                    out.append(param.text)
```

Comment:
Are there any memory considerations here? Would something like `param.text[cut-out-middle]` make sense?

Reply:
Not currently.